.github/workflows/conformance-gateway-api.yaml

```diff
@@ -311,6 +319,9 @@ jobs:
               -test.run "TestConformance" \
               -test.timeout=29m \
               -json \
+            | tee \
+              >(go-junit-report -parser gojson -code-owners="${CILIUM_CLI_CODE_OWNERS_PATHS}" -code-owners-prefix github.com/cilium/cilium/ -out "${JUNIT_PATH}") \
+              >(go run ./tools/testowners --code-owners="${CILIUM_CLI_CODE_OWNERS_PATHS}") \
```

Comment:
tbh, I am not sure how it works. I just triggered the test and hope it fails to see the result 😓